### PR TITLE
hubspot feedback

### PR DESCRIPTION
**Pull request by khanzadyan**

### CODE REVIEW

**Comment by khanzadyan** [comment]
@jhorbulyk got some feedback from Olga on some peculiarities of the component documentation. I thought you must be right person to look over my changes. Thanks in advance.

File: content/_components/hubspot/index.md [hunk 1]
```diff
@@ -24,43 +26,45 @@ A [{{site.data.tenant.name}}](http://www.{{site.data.tenant.name}}) component th
...
+During the credentials creation you can either select existing OAuth2 Client
+from the drop-down list or [choose to create a new OAuth2 Client](creating-oauth-app-for-hubspot)
+to use your created OAuth2 App in HubSpot.
 
-![Oauth2](img\client-exist.png)
+![Oauth2](img/hubspot-credentials-screen.png)
 
-* The **Additional parameters** input can be left empty.
-* In the **Scopes** input, the following value should be provided. (If you wish to only grant certain permissions, you can do so by removing them from this list.) 
+* Leave the **Additional parameters** input empty unless you have specifically configured this field in your HubSpot.
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_components/hubspot/index.md:38]: Here I'm assuming that additional parameters can be added on HubSopt side as required fields. So would like to cover that case as well. However, I'm not sure how.

File: content/_components/hubspot/index.md [hunk 2]
```diff
@@ -24,43 +26,45 @@ A [{{site.data.tenant.name}}](http://www.{{site.data.tenant.name}}) component th
...
+* Provide the following value in the **Scopes** input: `crm.objects.contacts.read`, `crm.objects.contacts.write`, `crm.schemas.contacts.read`, `crm.schemas.contacts.write`, `crm.objects.owners.read`. If you wish to grant certain permissions, you can do so by removing them from this list.
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_components/hubspot/index.md:39]: I can return it, my only concern is that the code section is not shown nicely and is confusing.

[the remark this answers is not in the document]
Comment (jhorbulyk) [reply content/_components/hubspot/index.md:39]: What is here seems to work fine with copy 
Screenshot from 2022-01-28 12-27-18
& paste. We can leave as is.

File: content/_components/hubspot/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhorbulyk) [reply content/_components/hubspot/index.md:38]: > I'm assuming that additional parameters can be added on HubSopt side as required fields.

I don't think this is the case in Hubspot.

File: content/_components/hubspot/index.md [hunk 3]
```diff
@@ -123,13 +123,13 @@ for fields `Attachment URL`. The component will throw an exception `process.unca
...
 
-Action to lookup object in Hubspot.
+Action to lookup object in HubSpot.
 Lookup Set will make sure all the items in the set should be there, otherwise throw an error.
 
 #### Configuration Fields
 
 *   **Object type** - Object type for lookup ("Companies", "Contacts", "Deals", "Line Items", "Tickets", "Quotes")
-*   **ID to Search On** - Identification to search object ("Hubspot Id" or "Email". "Email" is only for "Contacts" `Object Type`)
+*   **ID to Search On** - Identification to search object ("HubSpot Id" or "Email". "Email" is only for "Contacts" `Object Type`)
```
Comment (jhorbulyk) [inline content/_components/hubspot/index.md:132]:
```suggestion
*   **ID to Search On** - Identifier to search object ("HubSpot Id" or "Email". "Email" is only for "Contacts" `Object Type`)
```
Might as well make *identifier* usage consistent.

**Review by jhorbulyk — APPROVED** [review]

**Pull request merged**